templates/vault.systemd.erb

```diff
@@ -23,6 +23,7 @@ CapabilityBoundingSet=CAP_SYSLOG
 NoNewPrivileges=yes
 <% else -%>
 SecureBits=keep-caps
+AmbientCapabilities=CAP_IPC_LOCK
```

Comment:
Can you please update the associated tests in https://github.com/jsok/puppet-vault/blob/master/spec/classes/vault_spec.rb ?